src/nix/path-info.cc

```diff
@@ -34,12 +35,13 @@ static uint64_t getStoreObjectsTotalSize(Store & store, const StorePathSet & clo
  * @param showClosureSize If true, the closure size of each path is
  * included.
  */
-static json pathInfoToJSON(
+static std::pair<json, int> pathInfoToJSON(
```

Comment:
I would make this just a boolean, the "rendering" of the "did all paths substitute" should be separate from the info itself.